docs/design/namespaces.md

```diff
@@ -267,13 +267,13 @@ OpenShift creates a Namespace in Kubernetes
   "apiVersion":"v1",
   "kind": "Namespace",
   "metadata": {
-    "name": "development",
+    "name": "development"
   },
   "spec": {
-    "finalizers": ["openshift.com/origin", "kubernetes"],
+    "finalizers": ["openshift.com/origin", "kubernetes"]
   },
   "status": {
-    "phase": "Active",
+    "phase": "Active"
   },
   "labels": {
     "name": "development"
```

Comment:
I think we need to kill off the commas after "labels" too, right? (in all three places)